### PR TITLE
add scrutinizer config

### PR DESCRIPTION
**Pull request by janxious**

### CODE REVIEW

**Comment by janxious** [comment]
@markushausammann If you have other rules you want to add to scrutinizer, here is a place where you can do so.

File: src/DocRaptor.php [hunk 1]
```diff
@@ -279,4 +268,15 @@ public function fetchDocument($filename = false)
 
         return $filename ? true : $result;
     }
+
+
+    private function filterParam($allowedValues, $param, $paramType) {
```
Comment (janxious) [inline src/DocRaptor.php:282]: Should private functions be PHPDoc'ed?

Comment (markushausammann) [inline src/DocRaptor.php:282]: I'd say so.

**Comment by netshade** [comment]
:+1:

**Pull request merged**